sympy/core/tests/test_numbers.py

```diff
@@ -1749,3 +1749,41 @@ def test_Integer_precision():
     assert Float('1.0', precision=Integer(15))._prec == 15
     assert type(Float('1.0', precision=Integer(15))._prec) == int
     assert sympify(srepr(Float('1.0', precision=15))) == Float('1.0', precision=15)
+
+def test_numpy():
+    from sympy.utilities.pytest import skip
+    np = import_module('numpy')
+    def equal(x, y):
+        return x == y and type(x) == type(y)
+
+    if not np:
+        skip('numpy not installed.Abort numpy tests.')
+
+    assert Float(np.bool_(1)) is S(True)
```

Comment:
I'm not sure we should require `Float(bool)` to work, so no reason for it to work with `np.bool_`.